main.go

```diff
@@ -114,16 +123,10 @@ func main() {
 		// basePath. After that, logs will go only to stderr.
 		fileOnlyLogger.Printf("Dumped system information to: %s", basePath)
 
-		var stdout, stderr bytes.Buffer
-
-		cmd := exec.Command("tar", "-czf", basePath+".tar.gz", basePath)
-		cmd.Stdout = &stdout
-		cmd.Stderr = &stderr
-		if err := cmd.Run(); err != nil {
-			log.Printf("There was an error creating archiving the dumped data, so the dumped system information is stored unarchived in: %s", basePath)
+		if err := archive(basePath); err != nil {
+			log.Printf("Could not archive dump data, unarchived data in: %s", basePath)
```

Comment:
> I wonder would there be value in outputting the error that was returned from the archive function

I think there is no value. We could maybe store it in the dump, but there's not much point in printing "exec: tar not found" or something like that.

There should be no reason for people to be seeing this message if they installed the tool from the official RPM (that depends on tar, gzip).